src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,84 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
```

Comment:
I think this section is a good idea, but it's only Caddyfile examples. The Caddyfile is just a config adapter. This section should ideally be moved to an area of the docs specifically about the Caddyfile.

Reply:
As long as it's referenced from this doc for discovery, sure.

Most of the docs snippets I've seen are for Caddyfile configs, and as a new user I've found that most useful. I haven't tried the JSON configs yet, I'm not sure how much that differs but I have noticed it doesn't seem to be 1:1 equivalent, such as the JSON docs for settings of auto HTTPS feature has fields like `skip`.

Personally I'd like the tabbed example snippets like Traefik has, which lets you view the config in various supported formats. I was interested in the TOML adapter, but all I found was a link to a github repo with no documentation on how to use it, so I just stuck with Caddyfile for now.

If I move these, do you have any suggestions for how you want that handled? Is it going to become a large examples page for Caddyfile with many sections for different features over time, as a sort of reference? Or sub-pages for each feature?

Reply:
So remove this whole section and add it to the wiki, or should there also be a link to these wiki examples left somewhere in the doc?

Do I need to iterate the examples on the wiki, like the PR review here, or leave it for others to contribute improvements?

Reply:
> So remove this whole section and add it to the wiki

I think so. The wiki  is much easier for the community to help refine as well.

> should there also be a link to these wiki examples left somewhere in the doc?

You may absolutely link to wiki posts from the documentation, sure. :)

> Do I need to iterate the examples on the wiki, like the PR review here, or leave it for others to contribute improvements?

Why not both? 🤷‍♂️

Reply:
> Why not both?

I'm just not familiar with your wiki and process for that yet. I'm less likely to notice any notifications from other platforms. I'll move the examples to the wiki, and check on it depending how busy I am, otherwise I assume it's less stringent than a PR and the community is able to directly improve it as they see fit.